src/GPU/CalculateEwaldCUDAKernel.cu

```diff
@@ -86,15 +86,15 @@ void CallBoxReciprocalSetupGPU(VariablesCUDA *vars,
       imageSize);
   checkLastErrorCUDA(__FILE__, __LINE__);
 
-#ifndef NDEBUG
+//#ifndef NDEBUG
   // In the future maybe we could remove this for Nondebug?
   cudaMemcpy(sumRnew, vars->gpu_sumRnew[box],
              imageSize * sizeof(double),
              cudaMemcpyDeviceToHost);
   cudaMemcpy(sumInew, vars->gpu_sumInew[box],
              imageSize * sizeof(double),
              cudaMemcpyDeviceToHost);
-#endif
+//#endif
```

Comment:
In the current implementation, we need to copy sumRnew and sumInew from device to host, because the function 
```
void Ewald::BoxForceReciprocal(XYZArray const& molCoords,
                               XYZArray& atomForceRec,
                               XYZArray& molForceRec,
                               uint box)
```


uses the array in host.

Reply:
Ok.  The other two calls also?

Reply:
For all cases. If in a function, sumInew and sumRnew is changed, we need to copy them back to host